django/db/backends/sqlite3/operations.py

```diff
@@ -370,8 +372,27 @@ def subtract_temporals(self, internal_type, lhs, rhs):
             return 'django_time_diff(%s, %s)' % (lhs_sql, rhs_sql), params
         return 'django_timestamp_diff(%s, %s)' % (lhs_sql, rhs_sql), params
 
-    def insert_statement(self, ignore_conflicts=False):
-        return 'INSERT OR IGNORE INTO' if ignore_conflicts else super().insert_statement(ignore_conflicts)
+    def insert_statement(self, on_conflict=None):
+        if on_conflict == OnConflict.Ignore:
+            return 'INSERT OR IGNORE INTO'
+
+        elif on_conflict == OnConflict.Update and not DatabaseFeatures.supports_update_conflicts_with_target:
+            return 'INSERT OR REPLACE INTO'
+
+        return super().insert_statement(on_conflict=on_conflict)
+
+    def on_conflict_suffix_sql(
+        self, opts, fields, on_conflict=None, update_fields=None, unique_fields=None
+    ):
```

Comment:
We allow up to 119 characters per line and we haven't wrapped this in the other backends, so:

```suggestion
    def on_conflict_suffix_sql(self, opts, fields, on_conflict=None, update_fields=None, unique_fields=None):
```